DMC Scans: Fix selectors

sinkableShip opened the pull request:
Closes #2472

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

sinkableShip committed:
fix manga description selector

sinkableShip committed:
fix chapter feed selector

sinkableShip committed:
fix selector to script that contain pages link

sinkableShip committed:
bump overrideVersionCode

sinkableShip committed:
bump baseVersionCode

AwkwardPeak7 commented on src/en/dmcscans/src/eu/kanade/tachiyomi/extension/en/dmcscans/DMCScans.kt:
add `1, 3` ratelimit to this source too, as it was requsested by the website owner

vetleledaal commented on lib-multisrc/zeistmanga/build.gradle.kts:
```diff
@@ -2,4 +2,4 @@ plugins {
 }
 
-baseVersionCode = 9
+baseVersionCode = 10
```
Since there's no real change in the base class, it should be fine to leave this as-is

sinkableShip committed:
Revert "bump baseVersionCode"

This reverts commit 7b2cc6a937193130e7ce9532597dc4d23547985e.

sinkableShip committed:
change rate limit

sinkableShip committed:
change few selectors that affected because recent changes

vetleledaal commented on src/en/dmcscans/src/eu/kanade/tachiyomi/extension/en/dmcscans/DMCScans.kt:
```diff
@@ -10,7 +10,7 @@ import org.jsoup.Jsoup
 class DMCScans : ZeistManga("DMC Scans", "https://didascans.blogspot.com", "en") {
     override val client = super.client.newBuilder()
-        .rateLimit(2)
+        .rateLimit(1, 3)
```
Should add unit

vetleledaal commented on src/en/dmcscans/src/eu/kanade/tachiyomi/extension/en/dmcscans/DMCScans.kt:
```diff
@@ -23,6 +23,12 @@ class DMCScans : ZeistManga("DMC Scans", "https://didascans.blogspot.com", "en")
+    // =========================== Chapter Feed =============================
+
+    override val chapterFeedRegex = """clwd.\.run\(["'](.*?)["']\)""".toRegex()
+    override val scriptSelector = ".epcheck > script"
```
If selectors are constantly updated you need something more robust, like looking through every script tag and a more generic regex, or multiple regexes. Slightly different problem, but something semi-related can be seen [in this PR](https://github.com/keiyoushi/extensions-source/pull/2204).

sinkableShip committed:
change some selectors and other things

sinkableShip committed:
change calling super.pageListParse(response) into just copying the code

calling super.pageListParse(response) will cause java.lang.IllegalStateException: closed since response.asJsoup() will be called twice

sinkableShip committed:
fix indentation

sinkableShip committed:
add excluded category: web novel
